setup.cfg

```diff
@@ -25,37 +25,40 @@ keywords = pytorch, deep learning, machine learning, remote sensing, satellite i
 
 [options]
 install_requires =
-    einops<0.5
-    # fiona 1.5+ required for fiona.transform module
-    fiona~=1.5
+    # einops 0.3+ required for einops.repeat
+    einops>=0.3,<0.5
+    # fiona 1.8+ required for reading empty files
+    fiona>=1.8,<2
     # kornia 0.6.4+ required for kornia.contrib.compute_padding
     kornia>=0.6.4,<0.7
-    matplotlib<4
-    numpy<2
+    # matplotlib 3.3+ required for (H, W, 1) image support in plt.imshow
+    matplotlib>=3.3,<4
```

Comment:
Some of our datasets (idtrees?) are plotting (H, W, 1) images, can we just squash these so we can support older matplotlib?

Reply:
Should we keep track of these types of questions somewhere? Or resolve them here?

Reply:
I'm fine with either. If I get bored I'll try to fix these and support older versions. If I get busy I'll just merge this as is and deal with it another day.